dotCMS/src/main/java/com/dotmarketing/portlets/workflows/business/WorkflowAPI.java

```diff
@@ -192,10 +192,46 @@
 
 	public void reorderStep(WorkflowStep step, int order) throws DotDataException, AlreadyExistException;
 
+	/**
+	 * This is a legacy method for reorder
+	 * 
+	 * @deprecated On release 4.3, replaced by {@link #reorderAction(WorkflowAction, WorkflowStep, User, int)}
+	 * @param action WorkflowAction action you want to reorder, the getStepid has to be not empty and has to have the associated step to the action
+	 * @param order  int			Order for the action
+	 * @throws DotDataException
+	 * @throws AlreadyExistException
+	 */
+	@Deprecated
 	public void reorderAction(WorkflowAction action, int order) throws DotDataException, AlreadyExistException;
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1133)